include/swift/AST/ProtocolConformance.h

```diff
@@ -365,6 +373,7 @@ class NormalProtocolConformance : public ProtocolConformance,
     : ProtocolConformance(ProtocolConformanceKind::Normal, conformingType),
       ProtocolAndState(protocol, state), Loc(loc), ContextAndInvalid(dc, false)
   {
+    differenceAndStoreConditionalRequirements();
```

Comment:
Oh, so store it in the `ExtensionDecl` or something? I'm not sure where else this info should be stored.

Reply:
Or store it in the `NormalProtocolConformance`. Those are serialized, too.

Reply:
I haven't done this yet, but I will in a follow up: since the stdlib (etc.) isn't using this yet, it isn't necessary and I/you want this to be merged.